libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss

```diff
@@ -69,6 +69,10 @@
   }
 }
 
+.sky-text-editor-hint-text {
+  margin-top: var(--sky-margin-stacked-xs);
```

Comment:
I suggest removing this style and using `sky-margin-stacked-xs` directly on the element above the hint text element when hint text is specified.

Reply:
removed